src/bin/ldns.rs

```diff
@@ -6,21 +6,21 @@
 
 use std::process::ExitCode;
 
-use dnst::error::Error;
 use dnst::try_ldns_compatibility;
 
 fn main() -> ExitCode {
-    match run() {
+    let env = dnst::env::RealEnv;
+
+    let mut args = std::env::args_os();
+    args.next().unwrap();
+    let args =
+        try_ldns_compatibility(args).map(|args| args.expect("ldns commmand is not recognized"));
```

Comment:
Using `expect` here will presumably result in a rather ugly end user experience? Also, the LDNS command matching doesn't seem to be working, e.g.:

```
$ cargo build --release

$ ln -s target/release/dnst ldns-nsec3-blah

$ ./ldns-nsec3-blah 
Usage: ldns-nsec3-blah <COMMAND>

Commands:
  nsec3-hash  Prints the NSEC3 hash of a given domain name
  key2ds      Generate a DS RR from the DNSKEYS in keyfile
  help        Show the manual pages

Options:
  -h, --help     Print help
  -V, --version  Print version

Reply:
The ldns binary is for testing only (see comment at the top of the file)

Reply:
The second part, probably needs looking into

Reply:
That's expected in my opinion. The idea was to support the multicall behaviour for the ldns-tools, but anything else would be `dnst`. Is your suggestion to make it dependent on the `ldns-` prefix?

Reply:
And yes, the expect is because it's just an internal tool :)

Reply:
See: https://github.com/NLnetLabs/dnst/pull/33